app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,36 @@
+class BookingsController < ApplicationController
+  before_action :set_booking, only: [:show]
+  def index
+    @bookings = policy_scope(Booking).order(created_at: :desc)
+  end
+
+  def new
+    @booking = Booking.new
+    authorize @booking
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
+    @booking.user = current_user
+    authorize @booking
+    if @booking.save
+      redirect_to bookings_path
```

Comment:
shouldn't you redirect to the `show` of the new booking instead of the whole bookings page?
```suggestion
      redirect_to booking_path(@booking)
```